ToDoApp/ToDoApp/ToDoService.swift

```diff
@@ -1,108 +1,108 @@
+////
```

Comment:
is this file still used?

Reply:
No, it's not used anymore, and I have deleted it from the project.